clouddriver-kubernetes/src/integration/java/com/netflix/spinnaker/clouddriver/kubernetes/it/DeleteManifestIT.java

```diff
@@ -0,0 +1,157 @@
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class DeleteManifestIT extends BaseTest {
+
+  private static String APP_1_NAME = "app1";
+  private static String account1Ns;
+
+  @BeforeAll
+  public static void setUpAll() throws IOException, InterruptedException {
+    account1Ns = kubeCluster.createNamespace(ACCOUNT1_NAME);
+  }
+
+  public List<Map<String, Object>> buildStaticRequestBody(String manifestName, Boolean cascading) {
+    return KubeTestUtils.loadJson("classpath:requests/delete_manifest.json")
+        .withValue("deleteManifest.app", APP_1_NAME)
+        .withValue("deleteManifest.mode", "static")
+        .withValue("deleteManifest.manifestName", manifestName)
+        .withValue("deleteManifest.options.cascading", cascading)
+        .withValue("deleteManifest.location", account1Ns)
+        .withValue("deleteManifest.account", ACCOUNT1_NAME)
+        .asList();
+  }
+
+  public List<Map<String, Object>> buildDynamicRequestBody(
+      String manifestName, Boolean cascading, String cluster, String criteria, String kind) {
+    return KubeTestUtils.loadJson("classpath:requests/delete_manifest.json")
+        .withValue("deleteManifest.app", APP_1_NAME)
+        .withValue("deleteManifest.mode", "dynamic")
+        .withValue("deleteManifest.cluster", cluster)
+        .withValue("deleteManifest.criteria", criteria)
+        .withValue("deleteManifest.kind", kind)
+        .withValue("deleteManifest.manifestName", manifestName)
+        .withValue("deleteManifest.options.cascading", cascading)
+        .withValue("deleteManifest.location", account1Ns)
+        .withValue("deleteManifest.account", ACCOUNT1_NAME)
+        .asList();
+  }
+
+  public void deploy(final List<Map<String, Object>> manifest, String... objectNames)
+      throws InterruptedException {
+    List<Map<String, Object>> b =
+        KubeTestUtils.loadJson("classpath:requests/deploy_manifest.json")
+            .withValue("deployManifest.account", ACCOUNT1_NAME)
+            .withValue("deployManifest.namespaceOverride", account1Ns)
+            .withValue("deployManifest.moniker.app", APP_1_NAME)
+            .withValue("deployManifest.manifests", manifest)
+            .asList();
+    KubeTestUtils.deployAndWaitStable(baseUrl(), b, account1Ns, objectNames);
+  }
+
+  public void deployReplicaSets(byte amount) throws InterruptedException {
+    List<Map<String, Object>> m =
+        KubeTestUtils.loadYaml("classpath:manifests/replicaset.yml").asList();
+    for (byte i = 0; i < amount; i++) {
+      deploy(m, String.format("replicaSet nginx-v%03d", i));
+    }
+  }
+
+  @DisplayName(
+      ".\n===\n"
+          + "Given an existing static target\n"
+          + "When sending delete manifest request\n"
+          + "It should wait until the target is deleted\n===")
+  @Test
+  public void staticTarget() throws IOException, InterruptedException {
+    final String sn = "mysecret";
```

Comment:
There are missing comments like 
```
// ------------------------- given --------------------------
// ------------------------- when --------------------------
// ------------------------- then --------------------------
```
These are used to understand what is part of the actual tests and what is part of preparations or verification steps. These comments are also useful to correlate to the test case `DisplayName`.